upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -1381,6 +1381,9 @@ There you can find links to upgrade notes for other versions too.
 - fix symfony `dump()` function ([#1745](https://github.com/shopsys/shopsys/pull/1745))
     - see #project-base-diff to update your project
 
+- refactored `SelectToggle` component ([#1803](https://github.com/shopsys/shopsys/pull/1803))
+    the js class `ToggleOption` from framework node modules was removed
```

Comment:
Upgrade notes should be written in manner of orders people should do, so it should be like:
`ToggleOption` js class has been removed, update your code appropriately

Reply:
6e89aa0ef9dfcbcb9408ef4d441d1aecdc82a5c1